### PR TITLE
amp-ad type=tagon

### PR DESCRIPTION
**Pull request by Naxulanth**
Provides support for amp-ad type=tagon

### CODE REVIEW

File: ads/vendors/tagon.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
```
Comment (powerivq) [inline ads/vendors/tagon.js:2]: Nit: 2021

File: ads/vendors/tagon.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+<!---
+Copyright 2016 The AMP HTML Authors. All Rights Reserved.
```
Comment (powerivq) [inline ads/vendors/tagon.md:2]: Nit: 2021

**Review by powerivq — CHANGES_REQUESTED** [review]

**Comment by powerivq** [comment]
Please fix all the conflicts.

**Review by powerivq — APPROVED** [review]

**Comment by powerivq** [comment]
Please fix the error https://app.circleci.com/pipelines/github/ampproject/amphtml/10780/workflows/fc8ea9ee-7191-4491-9d73-76f62fcafd93/jobs/169252

**Comment by powerivq** [comment]
Tips: You can run `amp lint --fix` to fix the problems.

**Comment by powerivq** [comment]
Needs to resolve the merge conflict, or the tests won't run.

**Comment by powerivq** [comment]
Hey you will need to adopt the new patterns to have the new JS style error fixed. Sorry for the problems you are facing.

**Comment by Naxulanth** [comment]
Hey,

Is the build failing due to a problem on my side?

**Pull request merged**